app/controllers/ops_controller/settings/zones.rb

```diff
@@ -113,13 +115,30 @@ def zone_set_record_vars(zone, mode = nil)
     zone.settings ||= {}
     zone.settings[:proxy_server_ip] = @edit[:new][:proxy_server_ip]
     zone.settings[:concurrent_vm_scans] = @edit[:new][:concurrent_vm_scans]
+
+    zone.update_authentication({:windows_domain => {:userid => @edit[:new][:userid], :password => @edit[:new][:password]}}, {:save => (mode != :validate)})
```

Comment:
@DavidResende0 i don't think zone form has username/password fields, please check latest master.

Reply:
@DavidResende0 this code is removed in https://github.com/ManageIQ/manageiq-ui-classic/pull/7650, may be pull latest master into your branch?